diesel_cli/src/main.rs

```diff
@@ -1,5 +1,20 @@
+// Built-in Lints
+#![deny(warnings, missing_copy_implementations)]
+
+// Clippy lints
+#![cfg_attr(feature = "clippy", allow(unstable_features))]
```

Comment:
Though CLI should probably continue to allow things in release builds so that random lints added in future versions of Rust don't cause the binary to stop compiling.